src/oscontainer.py

```diff
@@ -152,29 +156,37 @@ def oscontainer_build(containers_storage, tmpdir, src, ref, image_name_and_tag,
                     subprocess.check_call(['/usr/lib/coreos-assembler/cp-reflink', entry.path, dest])
                 print(f"Copied in content from: {d}")
 
+        # We use /noentry to trick `podman create` into not erroring out
+        # on a container with no cmd/entrypoint.  It won't actually be run.
+        config = ['--entrypoint', '["/noentry"]',
+                  '-l', OSCONTAINER_COMMIT_LABEL + '=' + rev]
+        if ostree_version is not None:
+            config += ['-l', 'version=' + ostree_version]
+
+        base_pkgs = RpmOstree.db_query_all(r, rev, None)
+        for pkg in base_pkgs:
+            name = pkg.get_name()
+            if name in IMPORTANT_PACKAGES:
+                config += ['-l', f"com.coreos.rpm.{name}={pkg.get_evr()}.{pkg.get_arch()}"]
```

Comment:
And then this part is also related to https://github.com/ostreedev/ostree/issues/2251